prepare_benchmark

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python3
+
+
+import argparse
+import logging
+import os
+import sys
+
+import importlib.util
+
+from logging.config import fileConfig
+from swarm64_tpc_toolkit.streams import Streams, Benchmark
+
+fileConfig('configs/logging.ini')
+logger = logging.getLogger()
+
+if __name__ == '__main__':
+    py_version_info = sys.version_info
+    if py_version_info < (3, 6):
+        logger.error('Your python version does not match the requirements.')
+        sys.exit()
+
+    args_to_parse = argparse.ArgumentParser(argument_default=argparse.SUPPRESS)
+
+    args_to_parse.add_argument('--dsn', required=True, help=(
+        'The PostgreSQL DSN to connect to. Supply with DB, e.g. '
+        'postgresql://postgres@localhost/dbname'
+    ))
+
+    benchmarks = []
+    benchmarks_dir = 'benchmarks'
+    try:
+        for directory in os.listdir('benchmarks'):
+            base_dir = os.path.join(benchmarks_dir, directory)
+            benchmarks.append(Benchmark(name=directory, base_dir=base_dir))
+    except FileNotFoundError:
+        pass
+
+    benchmark_names = sorted([benchmark.name for benchmark in benchmarks])
+    args_to_parse.add_argument('--benchmark', required=True, choices=benchmark_names, help=(
+        'Which benchmark to run.'
+    ))
+
+    args_to_parse.add_argument('--scale-factor', type=int, default=None,
+                               required=('tpcds', 'tpch', 'ssb') in sys.argv, help=(
```

Comment:
sys.argv could be args.benchmark ?

Reply:
Not sure, it is not working atm anyways ;-)